app/controllers/sign_in_controller.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class SignInController < ApplicationController
```

Comment:
Should this be a sessions controller? so it can handle for sign in and sign out?

Reply:
Once DfE Signin has been added it be easier to address this as a whole sum, as there is already 2 other authentication mechanism that will need to be ratified and unified .